pkg/apis/migration/migration.go

```diff
@@ -325,22 +332,22 @@ func (volumeMigration *volumeMigration) registerVolume(ctx context.Context, volu
 		host = key
 		break
 	}
+	// Get vCenter
+	vCenter, err := vsphere.GetVirtualCenterManager(ctx).GetVirtualCenter(ctx, host)
+	if err != nil {
+		log.Errorf("failed to get vCenter. err: %v", err)
+		return "", err
+	}
+	// Connect to vCenter
+	err = vCenter.Connect(ctx)
```

Comment:
Why can this not be moved to VirtualCenter methods? We seem to be following this pattern where we rely on callers to make sure the connection is active/exists. I think we can reduce the complexity on the callers to do this by moving such things inside VirtualCenter methods.

Reply:
moved `vCenter.Connect(ctx)` to VirtualCenter methods.